tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -561,6 +565,75 @@ public function testAlterTableAutoIncrementIntToBigInt(
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+    public function testListTableColumnsOidConflictWithNonTableObject(): void
+    {
+        $versionParts      = [];
+        $wrappedConnection = $this->connection->getWrappedConnection();
+        assert($wrappedConnection instanceof ServerInfoAwareConnection);
+
+        $matchCount = preg_match('/^(?P<major>\d+)/', $wrappedConnection->getServerVersion(), $versionParts);
+        assert($matchCount > 0);
+
+        $majorVersion = $versionParts['major'];
+        if (version_compare($majorVersion, '10', '<')) {
```

Comment:
I don't think you understood my last comment about `version_compare()`. Have you tried if this works?

```suggestion
        if (version_compare($wrappedConnection->getServerVersion(), '10.0', '<')) {
```

Using `version_compare()` is a bit pointless if we still have to parse the server version.

Reply:
It worked.  Thanks for the clarification.